proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -21874,13 +21739,20 @@ crunches commit_time, tcb_sched_action
   for sc_tcbs_of[wp]: "\<lambda>s. P (sc_tcbs_of s)"
   (ignore: update_sched_context wp: crunch_wps)
 
+lemma cur_sc_tcb_bound_ready1:
```

Comment:
Check to see whether this can be removed, or whether this and the `cur_sc_tcb_bound_ready` can be generalised